readme.txt

```diff
@@ -244,8 +244,13 @@ Free public domain CC0 image.
 link-in-bio-image
 https://www.rawpixel.com/image/14262995
 
+Book Image - Created by Beatriz Fialho (@beafialho) using an image from rawpixel:
+Free public domain CC0 image.
+book-image.webp
+https://www.rawpixel.com/image/8799536
```

Comment:
This won't be necessary as we have it two times now. Let's move `book-image.webp` below `book-image-landing.webp`